src/modules/stateful_push_service/README.md

```diff
@@ -0,0 +1,16 @@
+stateful push service module
+===================
+
+The stateful push service module is made to abstract away from constructing
+the payloads required by the push get data function, but also to
+handle the bufferes required for push notifications.
+
+To use this module, add ${unabto_module_stateful_push_service_src} to your project CMakeLists.txt sources. When using push, add_definitions("-DUNABTO_PUSH_CALLBACK_FUNCTIONS") must also be in the project CMakeLists.txt.
```

Comment:
is it necessary to add it to definitions add_definitions("-DUNABTO_PUSH_CALLBACK_FUNCTIONS")